bin/incremental-diff-record.js

```diff
@@ -86,30 +87,51 @@ dyno.getItem({ Key: key }, function(err, data) {
     if (err) throw err;
     var dynamoRecord = data.Item;
 
-    s3.getObject(s3url, function(err, data) {
-        if (err && err.statusCode !== 404) throw err;
-        var s3data = err ? undefined : Dyno.deserialize(data.Body.toString());
-
-        console.log('DynamoDB record');
-        console.log('--------------');
-        console.log(dynamoRecord);
-        console.log('');
-
-        console.log('Incremental backup record (%s)', s3url.Key);
-        console.log('--------------');
-        console.log(s3data);
-        console.log('');
-
-        try {
-            assert.deepEqual(s3data, dynamoRecord);
-            console.log('----------------------------');
-            console.log('✔ The records are equivalent');
-            console.log('----------------------------');
-        }
-        catch (err) {
-            console.log('--------------------------------');
-            console.log('✘ The records are not equivalent');
-            console.log('--------------------------------');
-        }
-    });
+    s3Client.send(new GetObjectCommand(s3url))
```

Comment:
way too many callbacks being used here not related to your PR but it makes everything harder to read, maybe we can come back and use the await/async model